Makefile

```diff
@@ -0,0 +1,22 @@
+.PHONY: help
+help: ## Display this help.
+	@$(MAKE) help
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+
+##@ Testing:
+
+.PHONY: test
+test: ## Run tests using Helm chart-testing (ensure to run on dedicated test cluster)
```

Comment:
Maybe consider naming this target test-and-clean or something similar?

Reply:
the cleaning parts are only there to make it resilient, when testing I added the option --no-clean-up, and then you end up with lingering stuff, I then decided to keep those in there in case of failures to make it still work.

I'll refactor it to an undocumented cleaning task. As the clean shouldn't be relevant for the end-user.